src/components/modals/DarModal.js

```diff
@@ -43,71 +43,77 @@ const processResearchTypes = (researchTypes) => {
 };
 
 const requiresManualReview = (darDetails) => {
-  return darDetails.illegalBehavior ||
```

Comment:
dars sent from member and admin have a requiresManuelReiew property and none of the properties listed here, add a check so it works for bth dars from chairs and dars from admins and members